modules/dropdown/main/index.coffee

```diff
@@ -135,11 +140,29 @@ class Dropdown extends hx.EventEmitter
     if @isOpen() then @hide(cb) else @show(cb)
     this
 
+  dropdownContent: (dropdownContent, cb) ->
+    if arguments.length
+      setupDropdown = dropdownContentToSetupDropdown dropdownContent
+      @_.dropdownContent = dropdownContent
+      @_ = hx.shallowMerge @_, {
+        setupDropdown,
+        dropdownContent
+      }
+      @_.render cb
+    else
+      @_.dropdownContent
+
   show: (cb) ->
-    _ = @_
 
-    if not _.visible
-      _.visible = true
+    if not @_.visible
+      @_.visible = true
+      @render cb
+    this
```

Comment:
As render also checks for `@_.visible` could we just call `@render cb` outside the if statement and remove the explict `this` return?